### ISSUE TITLE
Spotify folders

### ISSUE THREAD

**Issue by anandado** [KRTirtho/spotube#862]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

I have lots of playlists on Spotify. So i have organized them in folders in order to find the types of playlists i want. Eg. "Relaxing". In Spotube there are no folders, so it becomes difficult to find eg. all the relax playlists. 

### Suggested solution

I wish the app will synchronize the folders from Spotify. Would also be good if Spotube can select folder to shuffle playback from all its contained playlists.

**Comment by modelair** [comment]
this app absolutely unusable while no folders here. 
it's a 3.4.1 for a second. three four one. 🤦‍♂️

**Comment by duncangeere** [comment]
Would also love to see folders. It would be nice if it synced it from Spotify, but I don't mind setting up the folders again myself once if necessary.

**Comment by psychosocial0** [comment]
I would also love to pin my playlists.

**Comment by guguss-31** [comment]
Why is there no answer to this question ? This feature is really useful !

**Comment by Pseudo-Sage** [comment]
Yes, I also made an enhancement request for this, but as that was duplicate it was deleted, but not answered. Please add this feature. God knows, if anyone will see this. It really helps to organize all 42 playlist and more by adding playlists to folders.

**Comment by codeswhite** [comment]
+1 to this.
@KRTirtho will it be feasible?

**Comment by KRTirtho** [comment]
Not anymore

**Comment by RokeJulianLockhart** [comment]
> https://github.com/KRTirtho/spotube/issues/862#issuecomment-3263865353

@KRTirtho, why?

**Comment by KRTirtho** [comment]
Please read the README.md at the repository root.

**Comment by RokeJulianLockhart** [comment]
@KRTirtho, shame to see. [^1] Apologies.

[^1]: [`github.com/KRTirtho/spotube/blob/0326630a1970bb63abff185667b67cbead050615/README.md#-spotube-is-banned-from-using-spotify-api-`][1]

[1]: https://github.com/KRTirtho/spotube/blob/0326630a1970bb63abff185667b67cbead050615/README.md#-spotube-is-banned-from-using-spotify-api-
